examples/raw.rs

```diff
@@ -13,66 +13,103 @@
 
 use futures::future::Future;
 use std::path::PathBuf;
-use tikv_client::*;
+use tikv_client::{raw::Client, Config, Key, KvPair, Result, Value};
 
-fn main() {
-    let config = Config::new(vec!["127.0.0.1:3379"]).with_security(
+const KEY: &str = "TiKV";
+const VALUE: &str = "Rust";
+const CUSTOM_CF: &str = "custom_cf";
+
+fn main() -> Result<()> {
+    // Create a configuration to use for the example.
+    // Optionally encrypt the traffic.
+    let config = Config::new(vec![
+        "192.168.0.101:3379", // Avoid a single point of failure,
```

Comment:
Mostly, it is better to use odd number od PD, so here we can use one or three for example.